src/css/custom.css

```diff
@@ -698,3 +698,53 @@ span.token.tag.script.language-javascript {
   height: 100%;
   border: 0;
 }
+
+.video-wrapper {
+  @apply p-7 md:p-28 bg-white relative overflow-hidden;
+}
+
+.video-wrapper::before {
+  content: "";
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-image: linear-gradient(90deg, #c5c5c5 1px, transparent 1px), linear-gradient(#c5c5c5 1px, transparent 1px);
+  background-size: 100px 50px;
+  background-position: -1px -1px;
+  opacity: 0.5;
+  max-width: 1595px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.video-container {
+  @apply relative w-full overflow-hidden pb-[56.25%] rounded-md md:rounded-3xl z-10;
+  box-shadow: 0px 4px 32px 0px #ffe933;
+}
+
+.video-background {
+  @apply absolute top-0 left-0 w-full h-full bg-cover bg-center;
```

Comment:
Let's not use `@apply` as its a deprecated feature.

https://www.xanthir.com/b4o00